api/RestfulService.php

```diff
@@ -53,7 +53,7 @@ public static function flush() {
 	 */
 	public static function set_default_curl_option($option, $value) {
 		Deprecation::notice('3.2', 'Use the "RestfulService.default_curl_options" config setting instead');
-		Config::inst()->update('RestfulService', 'default_curl_options', array($option => $value));
+		Config::inst()->update(get_called_class(), 'default_curl_options', array($option => $value));
```

Comment:
My comment pointing this out: https://github.com/silverstripe/silverstripe-framework/pull/4090#discussion_r28975400